### ISSUE TITLE
How to build a GraphConvModel with a customized training process ??

### ISSUE THREAD

**Issue by CHANG-Shaole** [deepchem/deepchem#3003]
## ❓ How to visualize the training process ?

Hello, the developer teams of DeepChem. I'm an engineer using AI technology, recently I was using your excellent work--**_deepchem_** to develop some customized models.

I want to build a GCN model, to do a **regression** task. Firstly I used the `GraphConvModel` which you had created and used the `fit` function to do the training process. But by using the fit function I can't see the whole training process.
In the other DL frameworks, such as `PyTorch`, I usually write some functions to **visualize the training process** (to show the accuracy & loss in each epoch ..maybe also use `tensorboard` tools). But by using the `fit` API, I can't do this. I believe I can do the same thing on the `deepchem` framework. But I'm not very familiar with it. That's why I want to open this issue.
Free answer for anyone who can solve it.

Thank you for such excellent work!

**Comment by peastman** [comment]
If you want to log information to Tensorboard, include the constructor arguments `tensorboard=True, log_frequency=N` when you create the model, where `N` is the frequency (in batches) at which it should log results.  See https://deepchem.readthedocs.io/en/latest/api_reference/models.html#torchmodel.  If you also want to report metrics for a validation set, use the `callbacks` argument to `fit()` to include a ValidationCallback.
